django_rq/tests/utils.py

```diff
@@ -5,15 +5,36 @@ def get_queue_index(name='default'):
     """
     Returns the position of Queue for the named queue in QUEUES_LIST
     """
-    queue_index = None
     connection = get_connection(name)
-    connection_kwargs = connection.connection_pool.connection_kwargs
+    connection_kwargs = dict(connection.connection_pool.connection_kwargs)
+
+    special_case_retry = False
+    if (retry := connection_kwargs.get('retry')) and retry.__class__.__eq__ == object.__eq__:
+        special_case_retry = True
+        del connection_kwargs['retry']
```

Comment:
Special casing can be removed if https://github.com/redis/redis-py/pull/3628 is accepted and released in a minor / point release and we don't need to support all previous releases (or the support for 6.0 is dropped, which will probably be in a long time)

Reply:
It seems like your PR was merged yesterday. https://github.com/redis/redis-py/pull/3628 . Should we just wait for redis-py 6.0.1 to be released before merging this in?

Reply:
Up to you, not sure how long that will be

Reply:
I think I might just exclude redis-py 6.0 from dependency. Will cut a release soon.

Reply:
I'll update this to not have the workaround then. I'm assuming you will have something like `redis-py!=6.0.0` in the dependency specifiers, so that when 6.0.1 is released this will work properly still.

Reply:
Yes, I merged this in just a few minutes ago: https://github.com/rq/rq/pull/2244

Reply:
Ok, I updated this PR for you